### ISSUE TITLE
Backend's GitHubActions::handlePagination(..) causing undefined error

### ISSUE THREAD

**Issue by braxtonhall** [ubccpsc/classy#353]
Observed throwing an undefined error when attempting to access `pText.match`.
Occurred when called by
```typescript
public async getTeamMembers(teamNumber: number): Promise<string[]>;
```
using
```
GH_API=https://api.github.com
```
on a team with one user.

Inspecting with the debugger shows that team members were successfully retrieved, but the code fumbles a bit later on before they could be returned.

<img width="582" alt="Screen Shot 2020-09-07 at 9 10 51 PM" src="https://user-images.githubusercontent.com/35436247/92432346-c1d61f80-f14e-11ea-99ce-8f963e04b3f1.png">

**Comment by andrewstec** [comment]
Curious about this. What is the extent of the bug? How did you come across this issue?

I see that you are using the Github.com API. Have you also seen this issue on Github Enterprise? My hunch is that this issue only occurs when integrated with Github.com because the tests pass when integrated with Github Enterprise.

**Comment by braxtonhall** [comment]
I came across it originally trying to log into the sdmm classy. All staff and admin are forced into student roles because of this bug, which I do think must have come from a change in the GitHub API.

310's classy is working just fine so I doubt it's a problem on Enterprise <sub><i>yet</i> </sub>

**Comment by rtholmes** [comment]
Caused by GitHub API change: https://developer.github.com/changes/2020-01-21-moving-the-team-api-endpoints/

Fixed in secapstone branch.

**Comment by rtholmes** [comment]
this was fixed again in PR #410 when we updated to the new teams api.
